tests/test-rest-pages-controller.php

```diff
@@ -47,6 +47,25 @@ public function test_get_items() {
 
 	}
 
+	public function test_get_items_private_filter_query_var() {
+		// Private query vars inaccessible to unauthorized users
+		wp_set_current_user( 0 );
```

Comment:
I wanted to test contributors here too, as they don't have the ability to edit pages, but I ran into #1892